benchmark/bench_RBKI/RBKI_runtime_breakdown.cc

```diff
@@ -136,7 +136,7 @@ int main(int argc, char *argv[]) {
     m_info.filename = argv[1];
     m_info.workspace_query_mod = 1;
     // Workspace query;
-    RandLAPACK::gen::mat_gen<double, r123::Philox4x32>(m_info, NULL, state);
+    RandLAPACK::gen::mat_gen<double>(m_info, NULL, state);
```

Comment:
Ah! Finally! This is a place where you _should_ specify the ``double`` template parameter. (Just making a note. You can mark this as resolved once you've read it.)

Reply:
Got it.